qt/qt_common/map_widget.cpp

```diff
@@ -337,6 +344,64 @@ void MapWidget::initializeGL()
   ASSERT(m_contextFactory == nullptr, ());
   if (!m_screenshotMode)
     m_ratio = devicePixelRatio();
+
+#if defined(OMIM_OS_MAC)
+  m_apiOpenGLES3 = false;
+#elif defined(OMIM_OS_LINUX)
+  {
+    QOpenGLFunctions * funcs = context()->functions();
+    const std::string vendor = QString::fromLatin1(
+      (const char*)funcs->glGetString(GL_VENDOR)).toStdString();
+    const std::string renderer = QString::fromLatin1(
+      (const char*)funcs->glGetString(GL_RENDERER)).toStdString();
+    const std::string version = QString::fromLatin1(
+      (const char*)funcs->glGetString(GL_VERSION)).toStdString();
+    const std::string glslVersion = QString::fromLatin1(
+      (const char*)funcs->glGetString(GL_SHADING_LANGUAGE_VERSION)).toStdString();
+    const std::string extensions = QString::fromLatin1(
+      (const char*)funcs->glGetString(GL_EXTENSIONS)).toStdString();
+
+    LOG(LINFO, ("Vendor:", vendor, "\nRenderer:", renderer,
+                "\nVersion:", version, "\nShading language version:\n", glslVersion,
+                "\nExtensions:", extensions));
+
+    if (context()->isOpenGLES())
+    {
+      LOG(LINFO, ("Context is LibGLES"));
+      // TODO: Fix the ES3 code path with ES3 compatible shader code.
+      m_apiOpenGLES3 = false;
+      constexpr const char* requiredExtensions[3] =
+        { "GL_EXT_map_buffer_range", "GL_OES_mapbuffer", "GL_OES_vertex_array_object" };
```

Comment:
We use 120 chars limit in the code. Do you have it set to 80?

```suggestion
      char constexpr * requiredExtensions[3] = { "GL_EXT_map_buffer_range", "GL_OES_mapbuffer", "GL_OES_vertex_array_object" };
```

Reply:
Also, note the reverse (RTL) order for const:
`char const * const` means const pointer to a const data.